### DISCUSSION TITLE
daisy-ext: Additional shades for DaisyUI colors

### DISCUSSION THREAD

**Discussion by nfaucher8** [saadeghi/daisyui#3258] [Show and tell]
Hey everyone, just wanted to share a library I have been working on that extends the DaisyUI color palette by adding shades to each of the DaisyUI colors. It does this by using [color-mix](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix) to combine existing DaisyUI colors and create new ones. This means it doesn't require any additional colors to be added to themes and works with existing themes without having to modify them.

![](https://raw.githubusercontent.com/nfaucher8/daisy-ext/HEAD/assets/primary-example.png)

I setup a demo site that shows the colors added and gives some examples of how the library could be used here:  [https://nfaucher8.github.io/daisy-ext-demo](https://nfaucher8.github.io/daisy-ext-demo)

You can check out the GitHub repository for the library here: [https://github.com/nfaucher8/daisy-ext](https://github.com/nfaucher8/daisy-ext)

If you want to try out the library for yourself you can get it from npm under the name `daisy-ext` [https://www.npmjs.com/package/daisy-ext](https://www.npmjs.com/package/daisy-ext)

Feel free to give it a try and hopefully someone might find it useful!

**Comment by saadeghi** [comment]
Wow this looks cool!  
Thanks for sharing!
